src/gui_main.py

```diff
@@ -0,0 +1,311 @@
+# native imports
+import atexit
+import os
+import sys
+import tkinter
+from tkinter import filedialog
+
+# third party imports
+import cript
+import eel
+import requests
+
+# my imports
+from python.excel_uploader_main import ExcelUploader
+
+
+class ExcelUploaderGUI:
+    def __init__(self):
+        # user input variables needed for Excel Uploader
+        self.host = None
+        self.api_key = None
+        self.project_name = None
+        self.collection_name = None
+        self.data_is_public = False
```

Comment:
We will not be allowing users to upload data as public from this tool, so this can be removed entirely.